public/data/DetailPage/DetailPageHeader/DetailPage_Accommodations_Data.json

```
@@ -2,8 +2,13 @@
   {
     "id": 1,
     "name": "마눌펜션",
+<<<<<<< HEAD
     "description": "애월! 넓은 잔디마당과 돌담~제주스러운 깨끗한 펜션",
     "region": "해변",
+=======
+    "description": "제주의 감성이 물씬! 돌담과 귤나무가 있는 독채 숙소!",
+    "detail_description": "안녕하세요!\n아름다운 산방산과 신비로운 언덕계곡이 흐르는\n아무도 간섭받지 않고 조용히 힐링하는 독채민박 입니다.\n\n[🍖 바베큐]\n- 이용요금 : 3만원\n- 제공품목 : 그릴, 숯, 불판, 가스자동토치, 부탄가스, 장갑, 집게, 가위 등\n -유의사항 : 고기 등 음식은 직접 준비 필요\n\n[📍 숙소 주변 관광지]\n화순해수욕장, 산방산, 중문관광단지, 용머리해안, 오설록, 카멜리아힐, 조각공원",
+>>>>>>> 00cceb7 (리뷰반영 완료)
```

```suggestion
    "description": "애월! 넓은 잔디마당과 돌담~제주스러운 깨끗한 펜션",
    "region": "해변",
    "description": "제주의 감성이 물씬! 돌담과 귤나무가 있는 독채 숙소!",
    "detail_description": "안녕하세요!\n아름다운 산방산과 신비로운 언덕계곡이 흐르는\n아무도 간섭받지 않고 조용히 힐링하는 독채민박 입니다.\n\n[🍖 바베큐]\n- 이용요금 : 3만원\n- 제공품목 : 그릴, 숯, 불판, 가스자동토치, 부탄가스, 장갑, 집게, 가위 등\n -유의사항 : 고기 등 음식은 직접 준비 필요\n\n[📍 숙소 주변 관광지]\n화순해수욕장, 산방산, 중문관광단지, 용머리해안, 오설록, 카멜리아힐, 조각공원",
```

conflict 해결 하고 남는 부분 이렇게 제거해주세요

---

해결하였습니다!